README.md

```diff
@@ -0,0 +1,138 @@
+[![License: BSD](
+    https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](
+        https://opensource.org/licenses/BSD-3-Clause)
+[![Build Status](
+    https://travis-ci.org/CVC4/CVC4.svg?branch=master)](
+        https://travis-ci.org/CVC4/CVC4)
+
+CVC4
+===============================================================================
+
+CVC4 is a tool for determining the satisfiability of a first order formula
+modulo a first order theory (or a combination of such theories).  It is the
+fourth in the Cooperating Validity Checker family of tools (CVC, CVC Lite,
+CVC3) but does not directly incorporate code from any previous version.
+
+CVC4 is intended to be an open and extensible SMT engine.  It can be used as a
+stand-alone tool or as a library.  It has been designed to increase the
+performance and reduce the memory overhead of its predecessors.  It is written
+entirely in C++ and is released under an open-source software license (see file
+[COPYING](https://github.com/CVC4/CVC4/blob/master/COPYING)).
+
+
+Website
+-------------------------------------------------------------------------------
+
+More information about CVC4 is available at:
+http://cvc4.cs.stanford.edu/
+
+Download
+-------------------------------------------------------------------------------
+
+The latest version of CVC4 can be found on GitHub:
+https://github.com/CVC4/CVC4
+
+Source tar balls and binaries for releases and latest stable builds of the
+[master branch](https://github.com/CVC4/CVC4) on GitHub can be
+found [here](http://cvc4.cs.stanford.edu/downloads).
+
+
+Build and Dependencies
+-------------------------------------------------------------------------------
+
+CVC4 can be built on Linux and macOS.  For Windows, CVC4 can be cross-compiled
+using Mingw-w64.
+
+For detailed build and installation instructions on these platforms,
+see file [INSTALL.md](https://github.com/CVC4/CVC4/blob/master/INSTALL.md).
```

Comment:
The file name is `INSTALL.md`.....

Reply:
It looks nicer without the extension.

Reply:
Well, that defeats the purpose, though, since it is necessary that users can identify the file.

Reply:
If the user is viewing it in a local repository, indeed. On github you have a link, so the name does not need to be the same. It'd be good to have two different files (one for github and another for the local repository), but since we don't (or won't), it's better as you made it.

Reply:
It is going to also serve as the install instructions for source releases and maintaining two files with the same content but different extensions is error prone since you always have to make sure to keep them in sync.